### ISSUE TITLE
Auto-detect edge in the browser metadata

### ISSUE THREAD

**Issue by kevinkuszyk** [wswebcreation/multiple-cucumber-html-reporter#25]
I have some e2e tests running in edge, but it's not picking up the browser in the metadata:

![image](https://user-images.githubusercontent.com/2734580/34937038-468c350c-f9db-11e7-9085-b78d5f3bd306.png)

and this in the json file:

```json
"metadata": {
  "browser": {
    "name": "microsoftedge",
    "version": "41.16299.15.0"
  }
}
```

It looks like the templates are looking for `edge`.  Would you take a PR which updates them to accept either string?

**Comment by wswebcreation** [comment]
Hi @kevinkuszyk 

Sorry for my late reaction, the notification of git of your question got lost in my emailbox. 

Yes, if you would like to provide a PR it would be very nice.

**Comment by kevinkuszyk** [comment]
Don't worry - it happens to me all the time.

Hopefully I'll get something over to you this week.

**Comment by wswebcreation** [comment]
Merged it and release version 1.4.2, tnx for the help!
